privacy/enforcement_test.go

```diff
@@ -264,7 +213,7 @@ func TestApply(t *testing.T) {
 		m.On("ScrubDevice", req.Device, test.expectedDeviceID, test.expectedDeviceIPv4, test.expectedDeviceIPv6, test.expectedDeviceGeo).Return(replacedDevice).Once()
 		m.On("ScrubUser", req.User, test.expectedUser, test.expectedUserGeo).Return(replacedUser).Once()
 
-		test.enforcement.apply(req, test.ampGDPRException, m)
+		test.enforcement.apply(req, m)
```

Comment:
Since we're not using `ampGDPRException` any longer, any reason to still keep it as part of the `testCase` struct?

Reply:
Nope, I'll remove. Good catch.